### ISSUE TITLE
Cannot find module 'fast-xml-parser'

### ISSUE THREAD

**Issue by ocelik94** [payloadcms/payload#5261]
### Describe the Bug

Hello,

I am new to this project and I tried to use your documentation to get started. When I followed the steps shown in the README.md of the starter template (website and blank) I keep getting following error in the npm run dev Step (or with docker-compose up -d).

```
payload-1   | warning payload > @swc/register@0.1.10: Use @swc-node/register instead
payload-1   | [2/4] Fetching packages...
payload-1   | [3/4] Linking dependencies...
payload-1   | warning "@payloadcms/bundler-webpack > swc-loader@0.2.6" has unmet peer dependency "@swc/core@^1.2.147".
payload-1   | warning "@payloadcms/bundler-webpack > swc-minify-webpack-plugin@2.1.1" has unmet peer dependency "@swc/core@^1.0.0".
payload-1   | warning "@payloadcms/richtext-slate > slate-react@0.92.0" has unmet peer dependency "react-dom@>=16.8.0".
payload-1   | warning "@payloadcms/richtext-slate > @faceless-ui/modal@2.0.1" has unmet peer dependency "react-dom@^16.8.0 || ^17.0.0 || ^18.0.0".
payload-1   | warning "@payloadcms/richtext-slate > @faceless-ui/modal > react-transition-group@4.4.5" has unmet peer dependency "react-dom@>=16.6.0".
payload-1   | warning "payload > html-webpack-plugin@5.5.3" has unmet peer dependency "webpack@^5.20.0".
payload-1   | warning "payload > swc-loader@0.2.3" has unmet peer dependency "webpack@>=2".
payload-1   | warning "payload > terser-webpack-plugin@5.3.9" has unmet peer dependency "webpack@^5.1.0".
payload-1   | warning Workspaces can only be enabled in private projects.
payload-1   | [4/4] Building fresh packages...
payload-1   | success Saved lockfile.
payload-1   | Done in 67.38s.
payload-1   | yarn run v1.22.19
payload-1   | $ cross-env PAYLOAD_CONFIG_PATH=src/payload.config.ts nodemon
payload-1   | [nodemon] 2.0.22
payload-1   | [nodemon] to restart at any time, enter `rs`
payload-1   | [nodemon] watching path(s): *.*
payload-1   | [nodemon] watching extensions: ts
payload-1   | [nodemon] starting `ts-node src/server.ts -- -I`
payload-1   | Error: Cannot find module 'fast-xml-parser'
payload-1   | Require stack:
payload-1   | - /home/node/app/node_modules/@aws-sdk/core/dist-cjs/index.js
payload-1   | - /home/node/app/node_modules/@aws-sdk/client-cognito-identity/dist-cjs/auth/httpAuthSchemeProvider.js
payload-1   | - /home/node/app/node_modules/@aws-sdk/client-cognito-identity/dist-cjs/index.js
payload-1   | - /home/node/app/node_modules/@payloadcms/plugin-cloud/dist/utilities/getStorageClient.js
payload-1   | - /home/node/app/node_modules/@payloadcms/plugin-cloud/dist/hooks/afterDelete.js
payload-1   | - /home/node/app/node_modules/@payloadcms/plugin-cloud/dist/plugin.js
payload-1   | - /home/node/app/node_modules/@payloadcms/plugin-cloud/dist/index.js
payload-1   | - /home/node/app/src/payload.config.ts
payload-1   | - /home/node/app/node_modules/payload/dist/config/load.js
payload-1   | - /home/node/app/node_modules/payload/dist/payload.js
payload-1   | - /home/node/app/node_modules/payload/dist/initHTTP.js
payload-1   | - /home/node/app/node_modules/payload/dist/index.js
payload-1   | - /home/node/app/src/server.ts
payload-1   |     at Function.Module._resolveFilename (node:internal/modules/cjs/loader:1134:15)
payload-1   |     at Function.Module._load (node:internal/modules/cjs/loader:975:27)
payload-1   |     at Module.require (node:internal/modules/cjs/loader:1225:19)
payload-1   |     at require (node:internal/modules/helpers:177:18)
payload-1   |     at Object.<anonymous> (/home/node/app/node_modules/@aws-sdk/core/dist-cjs/index.js:372:30)
payload-1   |     at Module._compile (node:internal/modules/cjs/loader:1356:14)
payload-1   |     at Module._extensions..js (node:internal/modules/cjs/loader:1414:10)
payload-1   |     at Object.require.extensions.<computed> [as .js] (/home/node/app/node_modules/ts-node/src/index.ts:1045:43)
payload-1   |     at Module.load (node:internal/modules/cjs/loader:1197:32)
payload-1   |     at Function.Module._load (node:internal/modules/cjs/loader:1013:12)
payload-1   | [nodemon] app crashed - waiting for file changes before starting...
```

### To Reproduce

1.  `npx create-payload-app@latest`
2. `docker compose up -d`

### Payload Version

2.0.0

### Adapters and Plugins

db-postgres

**Comment by AlessioGr** [comment]
Hey @ocelik94 I have followed your exacts steps and cannot reproduce this issue. I recommend trying a newer node version, as well as a fresh install, perhaps this was already fixed in a newer payload version in the mean-time. Also try running `yarn cache clear` before installing it again, as well as `yarn why fast-xml-parser` which should output something like this:

![CleanShot 2024-06-15 at 02 27 18@2x](https://github.com/payloadcms/payload/assets/70709113/250c8d1d-b442-4217-9726-999f3474b637)


If this still happens, please open a new issue with a linked minimal reproduction repository - then I can pull it down and test this on my local machine. Otherwise, I won't be able to reproduce and investigate this
